style/index.css

```diff
@@ -0,0 +1,245 @@
+@font-face {
+    font-family: Roboto-B;
+    src: url(..//fonts/Roboto-Bold.ttf);
+}
+
+@font-face {
+    font-family: Roboto-R;
+    src: url(..//fonts/Roboto-Regular.ttf);
+}
+
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+    list-style: none;
+}
+header {
+    position: fixed;
+    width: 100%;
+    height: 50px;
+    background: black;
+    background-image: url(../img/classy_fabric.png);
+    z-index: 1000;
+}
+body {
+    background: #ffffff;
+    background-image: url(../img/groovepaper.png);
+    height: 100%;
+    font-family: Roboto-R, Arial, sans-serif;
```

Comment:
You repeat this font-family a lot times below.
I guess, it worths appling this rule for the enitre page (let's say for the `<body>` element).